Optimization/README.md

```diff
@@ -0,0 +1,45 @@
+# Google Cloud Optimization for PHP
+
+> Idiomatic PHP client for [Google Cloud Optimization](https://cloud.google.com/optimization).
+
+[![Latest Stable Version](https://poser.pugx.org/google/cloud-optimization/v/stable)](https://packagist.org/packages/google/cloud-optimization) [![Packagist](https://img.shields.io/packagist/dm/google/cloud-optimization.svg)](https://packagist.org/packages/google/cloud-optimization)
+
+* [API documentation](http://googleapis.github.io/google-cloud-php/#/docs/cloud-optimization/latest/optimization/readme)
+
+**NOTE:** This repository is part of [Google Cloud PHP](https://github.com/googleapis/google-cloud-php). Any
+support requests, bug reports, or development contributions should be directed to
+that project.
+
+### Installation
+
+To begin, install the preferred dependency manager for PHP, [Composer](https://getcomposer.org/).
+
+Now to install just this component:
+
+```sh
+$ composer require google/cloud-optimization
+```
+
+Or to install the entire suite of components at once:
+
+```sh
+$ composer require google/cloud
```

Comment:
I know we talked about doing this all in one PR (I can take this one), but we should be removing this line here

Reply:
Added https://github.com/googleapis/google-cloud-php/issues/5273 to track this